README.md

```diff
@@ -1,2 +1,33 @@
 # jpo-asn-pojos
```

Comment:
suggestion: expand the README to include more in-depth project description, how to run tests, contributing, etc.

Reply:
Expanded the main README.